evidence/doordashcon/0002-doordashcon-08-2024.md

```diff
@@ -0,0 +1,64 @@
+# Evidence-0002: Retention at Rank I
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2024/08/21)                                                             |
+| **Submitted by**| Jesse Chejieh                                                 |
+
+
+## Member details
+
+- Matrix username: `@doordashcon/:matrix.org`
+- Polkadot address: `12zsKEDVcHpKEWb99iFt3xrTCQQXZMu477nJQsTBBrof5k2h`
+- Current rank: `1`
+- Date of initial induction: `2022/09/29`
+- Date of last report: `2024/05/04`
+- Area(s) of Expertise/Interest: `Frame & Runtime`
+
+
+## Reporting period
+
+- Start date: 2024/05/05
+- End date: 2024/08/24
+
+
+## Evidence
+I believe my contributions over the past period demonstrate my commitment to Polkadot's development and are in accordance with a Rank I member of The Fellowship.
+
+Grateful that I have been active in adding new features and collaborated with some higher ranked members of the fellowship, their guidiance has been invaluable to me.
+
+### Features
+Adding the [Secretary collective](https://github.com/polkadot-fellows/runtimes/pull/347) to the Collective System chain.
+
+### Mainteinance
+Refactoring the storage traits, with V1 pallet syntax removed the `generator` traits and method should be moved to `types` directory [[FRAME] Cleanup storage generator types](https://github.com/paritytech/polkadot-sdk/pull/5408).
+
+Members should not be required to manually call `bump` to begin the next salaly cycle, this adds a service task [Task(bump) Pallet Salary](https://github.com/paritytech/polkadot-sdk/pull/5163). 
+
+In the future I aim to take on more design decisions regarding FRAME.
+
+Still a lot to learn, overall I am enjoying my experience working with the Polkadot Ecosystem.
+
+
+## Voting record
+*Provide your voting record in relation to required thresholds for your rank.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities | Comments |
+|---|---|---|---|---|
+|III|70%   |100%  |I have voted on x out of xx referenda in which I was eligible to vote (i.e xx % voting activity). Out of xx referenda in which members of higher ranks were in complete agreement, I have voted in line with the consensus x times (i.e xx % voting agreement).  |*This is an example.* |
```

Comment:
```suggestion
|III|70%   |100%  |I have voted on x out of xx referenda in which I was eligible to vote (i.e xx % voting activity). Out of xx referenda in which members of higher ranks were in complete agreement, I have voted in line with the consensus x times (i.e xx % voting agreement).  | |
```

I think there should be real number in here?

Reply:
This is part of the template: https://github.com/polkadot-fellows/Evaluations/blob/main/0000-evidence-template.md#voting-record

Reply:
> I think there should be real number in here?

That row displays the example that comes with the evidence report template.
Jesse is Rank 1, so he shouldn't fill in anything for the rank 3 record.

It is probably best to remove this first row of the voting record table that has the example, @Doordashcon. ✍️